corehq/apps/locations/management/commands/remove_couch_loc_types.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand
+from dimagi.utils.couch.database import iter_docs
+from corehq.apps.domain.models import Domain
+
+
+class IterativeSaver(object):
+    """
+    Bulk save docs in chunks.
+
+        with IterativeSaver(db) as iter_db:
+            for doc in iter_docs(db)
+                iter_db.save(doc)
+    """
+    def __init__(self, database, chunksize=100):
+        self.db = database
+        self.chunksize = chunksize
+
+    def __enter__(self):
+        self.to_save = []
+        return self
+
+    def commit(self):
+        self.db.bulk_save(self.to_save)
+        self.to_save = []
+
+    def save(self, doc):
+        self.to_save.append(doc)
+        if len(self.to_save) >= self.chunksize:
+            self.commit()
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if self.to_save:
+            self.commit()
+
+
+class Command(BaseCommand):
+    help = "(2015-04-02) Delete the location_types property on Domain"
```

Comment:
It's a one-time migration file, so I stuck a date on it so people know it can be deleted later.  I could probably be more explicit about that.